src/site/content/en/learn/css/specificity/index.md

```diff
@@ -0,0 +1,363 @@
+---
+title: Specificity
+description:
+authors:
+  - andybell
+date: 2021-04-02
+tags:
+  - css
+---
+
+# Specificity
+
+Suppose that you're working with the following HTML and CSS:
+
+```html
+<button class="branding">Hello, Specificity!</button>
+```
+
+```css
+button {
+  color: red;
+}
+
+.branding {
+  color: blue;
+}
+```
+
+There's two competing rules here.
+One will color the button red and the other will color it blue.
+Which rule gets applied to the element?
+Understanding the CSS specification's algorithm about specificity
+is the key to understanding how CSS decides between competing rules.
+
+Specificity is one of the four distinct stages of the cascade,
+which you learned about in the last module.
+
+<figure class="w-figure">
+{% Codepen {
+  user: 'web-dot-dev',
+  id: 'YzNKMXm'
+} %}
+</figure>
+
+## Specificity scoring
+
+Each selector rule gets a scoring.
+You can think of specificity as a total score and each selector type earns points towards that score.
+The selector with the highest score wins.
+
+With specificity in a real project,
+the balancing act is making sure the CSS rules you expect to apply, actually *do apply,*
+while generally keeping scores low to prevent complexity.
+The score should only be as high as we need it to be,
+rather than going for the highest score possible.
+In the future, some genuinely more important CSS might need to be applied.
+If you go for the highest score, you'll make that job hard.
+
+## Scoring each selector type
+
+Each selector type earns points.
+You add all of these points up to calculate a selector's overall specificity.
+
+### Universal selector
+
+A universal selector (`*`) has **no specificity** and gets **0 points**.
+This means that any rule with 1 or more points will override it
+
+```css
+* {
+	color: red;
+}
+```
+
+### Element or pseudo-element selector
+
+An element (type) or pseudo-element selector gets **1 point of specificity** .
+
+### Type selector
+
+```css
+div {
+	color: red;
+}
+```
+
+### Pseudo-element selector
+
+```css
+::selection {
+	color: red;
+}
+```
+
+### Class, attribute or pseudo-class selector
+
+A class or pseudo-class selector gets **10 points of specificity**.
+
+Class selector:
```

Comment:
```suggestion
#### Class selector
```

Reply:
Then these ones could be an h4, as they are sub-sub heads because there is a h3 which is https://deploy-preview-5019--web-dev-staging.netlify.app/learn/css/specificity/#class-attribute-or-pseudo-class-selector